# HG changeset patch
# User Rik <rik@octave.org>
# Date 1343798668 25200
#      Tue Jul 31 22:24:28 2012 -0700
# Node ID c53c28c7c811e24fcc5bc8f808743bda3cbde5a3
# Parent  1feefc5e56304d4c8d9f4f037d2c959aa5363efc
configure.ac: Comment out unneeded special build flags for Sun C++ compiler.

* configure.ac: Comment out unneeded special build flags for Sun C++ compiler.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -482,27 +482,28 @@ if test "$cross_compiling" = yes; then
   BUILD_LDFLAGS=""
   BUILD_EXEEXT=""
 else
   BUILD_CC='$(CC)'
   BUILD_CFLAGS='$(CFLAGS)'
   BUILD_CXX='$(CXX)'
   BUILD_CXXFLAGS='$(CXXFLAGS)'
   BUILD_LDFLAGS='$(LDFLAGS)'
-  case "$canonical_host_type" in
-    sparc-sun-solaris2*)
-      if test "$GCC" != yes; then
-        ## FIXME: gendoc.cc is no longer used for building documentation.
-        ##        Can this test be eliminated? 
-        ## The Sun C++ compiler never seems to complete compiling
-        ## gendoc.cc unless we reduce the optimization level...
-        BUILD_CXXFLAGS="-g -O1"
-      fi
-    ;;
-  esac
+  ## 2012/07/31: Commented out special build requirements
+  ## for Sun compiler now that gendoc.cc is no longer part of build.
+  ##################################################################
+  #case "$canonical_host_type" in
+  #  sparc-sun-solaris2*)
+  #    if test "$GCC" != yes; then
+  #      ## The Sun C++ compiler never seems to complete compiling
+  #      ## gendoc.cc unless we reduce the optimization level...
+  #      ## BUILD_CXXFLAGS="-g -O1"
+  #    fi
+  #  ;;
+  #esac
   BUILD_EXEEXT='$(EXEEXT)'
 fi
 
 AC_ARG_VAR(BUILD_CC, [build system C compiler (used if cross compiling)])
 AC_ARG_VAR(BUILD_CFLAGS, [build system C compiler flags (used if cross compiling)])
 AC_ARG_VAR(BUILD_CXX, [build system C++ compiler (used if cross compiling)])
 AC_ARG_VAR(BUILD_CXXFLAGS, [build system C++ compiler flags (used if cross compiling)])
 AC_ARG_VAR(BUILD_LDFLAGS, [build system C++ compiler link flags (used if cross compiling)])
